Readme update (#24)
alat-rights/alana-utilities: docs/README.md

  - `alana.remove_xml` to strip certain XML tag-enclosed content from a string (along with the tags). This is primarily intended to get rid of "<reasoning>...</reasoning>" strings. ⚠️ Regex parsing of XML may be unreliable!
- A bunch of aliases (Try: `alana.few_shot`, `alana.n_shot`, or `alana.xml`)

## Contributing

> A big welcome and thank you for considering contributing to alana-utilities open source project!

Reviewer: ```suggestion
Welcome! Thank you for considering making a contribution to `alana-utilities`.
```

### Getting started

> For changes that address core functionality or would require breaking changes (e.g. a major release), it's best to open an Issue to discuss your proposal first. This is not required but can save time creating and reviewing changes.

Reviewer: ```suggestion
If you'd like to make a major refactor or breaking change, please open an [Issue](https://github.com/alat-rights/alana-utilities/issues) first. This way, I can review it to determine if it is within scope for this project.

In general, there are no guarantees that your pull requests will be merged into the library. `alana-utilities` is essentially a hobby project, and there is currently only one maintainer.
```

In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr).

### Style and formating
We are using [Black](https://github.com/psf/black) for formating and [PyRight](https://github.com/microsoft/pyright) for type checking.

> Before pushing your code and making PR make sure that you have run `black .` on your code, otherwise it will fail.

Reviewer: ```suggestion
Before submitting a pull request, please run `black .` on your code. This saves us from failed CI runs.
```

> To install and run black use this commands:

Reviewer: ```suggestion
To install and run `Black`:
```

```commandline
  $ pip install black
  $ black .
```

> To keep types in code consistent:

Reviewer: ```suggestion
To install and run `PyRight`:
```

  - `alana.remove_xml` to strip certain XML tag-enclosed content from a string (along with the tags). This is primarily intended to get rid of "<reasoning>...</reasoning>" strings. ⚠️ Regex parsing of XML may be unreliable!
- A bunch of aliases (Try: `alana.few_shot`, `alana.n_shot`, or `alana.xml`)

## Contributing
Welcome! Thank you for considering making a contribution to `alana-utilities`.

### Getting started
If you'd like to make a major refactor or breaking change, please open an [Issue](https://github.com/alat-rights/alana-utilities/issues) first. This way, I can review it to determine if it is within scope for this project.

> In general, there are no guarantees that your pull requests will be merged into the library. `alana-utilities` is essentially a hobby project, and there is currently only one maintainer.

Reviewer: ```suggestion
There are no guarantees that your pull requests will be merged into the library. `alana-utilities` is essentially a hobby project, and there is currently only one maintainer.

```

In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr).

### Style and formating

> We are using [Black](https://github.com/psf/black) for formating and [PyRight](https://github.com/microsoft/pyright) for type checking.

Reviewer: ```suggestion
We are using [Black](https://github.com/psf/black) for formatting and [PyRight](https://github.com/microsoft/pyright) for type checking.
```

Before submitting a pull request, please run `black .` on your code. This saves us from failed CI runs.

> To install and run `Black`:

Reviewer: ```suggestion

To install and run `Black`:
```

```commandline
  $ pip install black
  $ black .
```

> To install and run `PyRight`:

Reviewer: ```suggestion

To install and run `PyRight`:
```
